modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,6 +19,8 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Catalog | Chains | Hub | {pac} | |
 
+|1.9 | 0.41.x | 0.22.x | 0.28.x | NA | 0.13.x (TP) | 1.11.x | 0.15.x (GA) | 4.11, 4.12, 4.13 | GA
```

Comment:
Should 4.13 appear in 4.12/4.11 documentation?

Reply:
@Preeticp, Could you please suggest whether we need to include version 4.13 in this matrix table or not? I included 4.13 based on this reference link: https://spaces.redhat.com/pages/viewpage.action?spaceKey=SRVKP&title=Pipeline+Component+Matrix

Reply:
+1 I would leave out 4.13 as it is not yet out. @sounix000 can you please chime in as well.

Reply:
In the last release I wrote it as "planned". https://docs.openshift.com/container-platform/4.11/cicd/pipelines/op-release-notes.html#compatibility-support-matrix_op-release-notes

Reply:
Added `planned` against 4.13 in the matrix table.